docs/dev/antora.yml

```diff
@@ -17,8 +17,6 @@ runtime:
 
 asciidoc:
   attributes:
-    review: ''
-    todo: ''
```

Comment:
Isn't this just the local dev view? Not included in published docs?

Reply:
Hmm, correct. I missed the fact that it was in `dev` path. 

I will check it once again. There must be some other properties or we are using dev to publish the docs, because we do have the TODOs showing up in docs.kalix.io.

Reply:
And indeed, we want to see the TODOs when building locally and my change remove it completely.